moldesign/models/amber2lammps.py

```diff
@@ -0,0 +1,945 @@
+#! /usr/bin/evn python2
```

Comment:
Let's move this file to `moldesign/external` and import it from there, just for consistency